spec/features/indexing_custom_component_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'Indexing Custom Component', type: :feature do
+  let(:components) do # an Array<Arclight::CustomComponent>
+    options = { component: Arclight::CustomComponent }
+    ENV['SOLR_URL'] = 'http://127.0.0.1:8983/solr/blacklight-core'
```

Comment:
The fact that we have this hard-coded in the rake task was curious to me.  Is there a way we can set this w/o hard-coding the solr URL (and perhaps we can use the same pattern in other places).